### PR TITLE
Changes to 'Building Java Cloud Applications with Micronaut and OCI' workshop

### PR DESCRIPTION
**Pull request by kamryn-v**
1. Added screenshots
2. Corrected typos

### CODE REVIEW

File: developer-library/micronaut-oci-atp/helidon/helidon.md [hunk 1]
```diff
@@ -30,11 +30,7 @@ Download the native image binary (this is needed for deployment to cloud)
 
 Run the native image (if you are using a Linux environment):
 
-```bash
-wget -O helidon-mp-service https://objectstorage.us-phoenix-1.oraclecloud.com/n/toddrsharp/b/micronaut-lab-assets/o/helidon-mp-service
-chmod +x helidon-mp-service
-./helidon-mp-service
-```
```
Comment (graemerocher) [inline developer-library/micronaut-oci-atp/helidon/helidon.md:37]: Why were these lines deleted?

File: developer-library/micronaut-oci-atp/helidon/helidon.md [hunk 2]
```diff
@@ -46,7 +42,7 @@ This will open the application on port `8081`
 We can verify this application works by exercising the following endpoints:
 
 - `curl -i http://localhost:8081/vaccinated/Dino` - this is our "business" endpoint and should return `true`
-- `curl -i http://localhost:8081/health` - returns health status (MicroProfile health format)
+- `curl -i http://localhost:8080/health` - returns health status (MicroProfile health format)
```
Comment (graemerocher) [inline developer-library/micronaut-oci-atp/helidon/helidon.md:45]: Why was this port changed?

File: developer-library/micronaut-oci-atp/deploy/deploy.md [hunk 3]
```diff
@@ -47,7 +50,8 @@ In this lab you will:
 
 4. Push Helidon native image to the VM:
 
-    ```bash
+    ```
+    bash
```
Comment (graemerocher) [inline developer-library/micronaut-oci-atp/deploy/deploy.md:54]: I think the word `bash` can just be removed for all these snippets

**Review by graemerocher — CHANGES_REQUESTED** [review]
hi @kamryn-v thanks for the PR. Some changes seem to modify code snippets, was this intentional?

**Comment by graemerocher** [comment]
@kamryn-v could you alter to include only your changes?

**Comment by kamryn-v** [comment]
Yes, I am pushing the changes to Github to my personal account now.

**Review by kaymalcolm — COMMENTED** [review]
Looks good Kamryn!  I made a few edits.  @graemerocher are you good with this iteration?

**Comment by graemerocher** [comment]
Yes looks good

**Review by kaymalcolm — APPROVED** [review]

**Pull request merged**